packages/wasm/test/integration.test.js

```diff
@@ -3,40 +3,46 @@
 const HOST = `http://localhost:${process.env.PORT}`;
 
 describe('Wasm', () => {
-  it('captured exception should include modified frames and debug_meta attribute', async () => {
-    await page.goto(HOST);
-    const event = await page.evaluate(async () => {
-      return window.getEvent();
-    });
+  // TODO: This is a quick and dirty way to test the minified browser bundle - `min-bundle.html` is an exact replica of
+  // `index.html` save the browser bundle `src` value. In the long run, we should rig it so just the bundle can be
+  // passed in. (Or, once the new bundling process is nailed down, stop testing against the minified bundle, since
+  // that's not really what this test is for.)
+  ['index.html', 'min-bundle.html'].forEach(pagePath =>
```

Comment:
I was going to suggest dynamically generating these using a template, but we should wait till the new bundling process is nailed down.

Reply:
Yup, that's exactly what I meant by passing the bundle in. If we decide to keep testing against both, a template is indeed probably the easiest way to do that. I'm not 100% convinced we'll even need to/want to separately test against the minified bundle once we've solved bundling, but that's a bridge we can cross when we get there.